libfreerdp/core/gateway/arm.c

```diff
@@ -56,18 +56,26 @@
 #include <cjson/cJSON.h>
 #endif
 
+#include <time.h>
```

Comment:
Not sure where we now use functions from time.h

Reply:
no, removed.